README.md

```diff
@@ -1632,3 +1638,4 @@ Come on, let me see what is happening here!
 ![cat in glasses](https://media4.giphy.com/media/VbnUQpnihPSIgIXuZv/giphy-downsized.gif)
 +
 ---
+
```

Comment:
Good job! 
fyi: You can setup your code editor to add empty line at the end of the file by itself